src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+* {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
```

Comment:
remove position: fixed;, why you even need it here?
add horizontal padding to header (50 px for each side), check the design

Reply:
I've used fixed for bottom line under the "APPLE", but it also works with Sticky, I'll use it